packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -448,6 +448,40 @@ export function createLiveblocksContext<
     }, []);
   }
 
+  function useInboxNotificationThread(
+    inboxNotificationId: string
+  ): ThreadData<BaseMetadata> {
+    const selector = useCallback(
+      (state: CacheState<BaseMetadata>) => {
+        const inboxNotification = state.inboxNotifications[inboxNotificationId];
+
+        if (inboxNotification === undefined) {
+          throw new Error(
+            `Inbox notification with id "${inboxNotificationId}" not found`
+          );
+        }
+
+        const thread = state.threads[inboxNotification.threadId];
```

Comment:
Now that we have custom notifications that don't have a `threadId` property, does this work? Did you update your branch to get the latest state of the 1.12 branch?

Reply:
Indeed, it didn't work the updated typing. I've added an error if the notification kind is not "thread".